### ISSUE TITLE
ListGroup flashes with unsorted content when re-opened and needs to sort

### ISSUE THREAD

**Issue by barmac** [bpmn-io/properties-panel#107]
__Describe the Bug__

When I create multiple elements in a list group, it flashes with unsorted content when I re-open. 

https://user-images.githubusercontent.com/28307541/136554150-beeb9c95-1d29-403e-b983-d1be1a6f52bf.mp4

__Steps to Reproduce__

1. Create multiple elements
2. Name them from [c, b, a] to force sorting
3. Close and open

__Expected Behavior__

No flash. It should be sorted right away

**Comment by smbea** [comment]
I can't reproduce this. It appears sorted when re-opening. Maybe we fixed this along the way?

**Comment by barmac** [comment]
Let's close it then.
